src/azure-iot-edge-installer.sh

```diff
@@ -20,7 +20,8 @@ download_bash_script() {
         log_info "downloading '%s'" $file_name
 
         # attempt to download to a temporary file.
-        wget $url_text -q -O $tmp_file
+        #wget $url_text -q -O $tmp_file
+		cp ../$file_name .
```

Comment:
- I assume this is only temporary :-)
- Please merge with 'main' and use:
cp ../$TOPDIR/$file_name .
- nit: replace tabs with spaces :-)